.circleci/config.yml

```diff
@@ -817,11 +817,11 @@ jobs:
     executor: reuse_tool
     steps:
       - restore_repo
-      - setup_authenticated_remote_docker
+      - run: reuse lint
       - run:
-          name: Check REUSE licensing status
-          command: |
-            ./bin/add-license-headers.sh --lint-only
+          when: on_fail
+          name: Install dependencies
+          command: apk add curl jq
```

Comment:
Why is stuff installed on failure? 🤔

Reply:
The silly Slack orb needs `curl` and `jq`installed and those parts only run on failure -> no need to waste time otherwise. The Slack orb has auto-install for those deps during its run for most other systems (even Homebrew!) but for some reason not for `apk` (although it even has a branch for it).